information.txt

```diff
@@ -24,18 +24,22 @@ Overall evaluation: 0 (borderline paper)
 The paper advertises the cloud platforms as efficient support in teachning the machine learning methods for large data mining.
 The concept is interesting, however the major revision of the paper is necessary for making it ready for presentation at the conference.
 
+<<<CHRIS NOTE - I feel like most of this reviewer's comments should be addressed in the introduction. They are not details / implementation questions, but more "motivation and why is this useful?" questions. I tried
+to separate out the intro into a shorter "proper" intro section, and a second more long-winded one, to make
+these points more clearly >>>
+
 1) I have a feeling that in fact the authors would like to compare the clouds with traditional work with repositories, but I cannot see any significant defferences amoung those two tevhnologies in the paper. The authors use the cloud mainly to give for the students the remote access to the teaching material and data and an opportunity to upload their work into cloud.
-It is unclear for me what kind of other services are used.
+It is unclear for me what kind of other services are used.  <--- emphasized the fact that students don't need to download anything new onto their computer, they only go to a website.
 
-2) The author mention the requirement of Knowledge in cloud-computing infrastructure as the drawback of the cloud approach. I cannot understand it. The main aim of using the cloud is to make such infrastructure 'invisible' to the end users (students). Why do they need such knowledge?
+2) The author mention the requirement of Knowledge in cloud-computing infrastructure as the drawback of the cloud approach. I cannot understand it. The main aim of using the cloud is to make such infrastructure 'invisible' to the end users (students). Why do they need such knowledge?  <--- Tried to clarify language in the intro saying only one person needs any knowledge of cloud infrastructure.
 
-3) The conclusions section should be moved to the end of the paper
+3) The conclusions section should be moved to the end of the paper  <--- I kinda wanna push back on this. I like the methods being last because it's more of an appendix. what do people think?
```

Comment:
I think that we can split what we now call "conclusion" into two different parts. The first would be a one-paragraph section 2.4, which would serve as an interim summary. The other would be the part starting with "the purpose of this manuscript", and we can move it to the end. The paper is short enough, so I don't think it's too fundamental that the last part be the more detailed methods section. Anyway, some readers might want something at the end that tells them what the  paper actually concluded.